docs/installation/installing_gekko_on_windows.md

```diff
@@ -46,4 +50,32 @@ Your browser should automatically open with the UI. If it doesn't, manually brow
     
 ### Stopping Gekko
 
-In the command line hold `ctrl` + `c`.
\ No newline at end of file
+In the command line hold `ctrl` + `c`.
+
+## Common Problems
+
+If you get an error like one of these, try the suggested action.
+
+#### Failed to locate CL.exe:
+
+`
+C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppBuild.targets(366,5): warning MSB8003: Could not find WindowsSDKDir variable from the registry. TargetFrameworkVersion or PlatformToolset may be set to an invalid version number. 
+`
+
+`
+TRACKER : error TRK0005: Failed to locate: "CL.exe". The system cannot find the file specified. 
+`
+
+`
+gyp ERR! build error
+gyp ERR! stack Error: C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe failed with exit code: 1
+gyp ERR! stack at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:269:23)
+gyp ERR! stack at ChildProcess.emit (events.js:110:17)
+gyp ERR! stack at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
+gyp ERR! System Windows_NT 6.3.9600
+gyp ERR! command "node" "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" "rebuild"
+`
+
+#### Suggested action:
```

Comment:
I was using the LTS version of Nodejs when I encountered this. Although, I used `npm install --only=production`. I can try to reproduce it if you want.